presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDatabaseMetaData.java

```diff
@@ -94,6 +100,37 @@ public void testPassEscapeInMetaDataQuery()
         assertContains(query, "_t' ESCAPE '", "Metadata query does not contain ESCAPE");
     }
 
+    @Test
+    public void testGetTypeInfo()
+            throws Exception
+    {
+        DatabaseMetaData metaData = connection.getMetaData();
+        ResultSet typeInfo = metaData.getTypeInfo();
+        try (ResultSet rs = statement.executeQuery("SELECT * FROM system.jdbc.types ORDER BY DATA_TYPE")) {
```

Comment:
I don't think this is a very useful test, since the `getTypeInfo()` method is basically executing an identical query. How about spot checking these results for a few types? See `assertColumnSpec()` in `TestPrestoDriver`.